src/portal/fixtures/list-organizations.json

```diff
@@ -0,0 +1,90 @@
+{
```

Comment:
We haven't done this pattern yet! I wanted to test what the SDK does with the data it receives and confirm that it drops it in an acceptable data structure. Previous similar tests were just testing that we formed the correct request to the API.

It's super easy to generate these responses in Postman or, in this case, I just grabbed it from the Ruby SDK VCR cassettes.

Reply:
I like it 👍

Reply:
I like it 👍